tests/test-cases/Group1-Docker-Commands/TestCases.md

```diff
@@ -29,4 +29,22 @@ Group 1 - Docker Commands
 [Test 1-13 - Docker Version](1-13-Docker-Version.md)
 -
 [Test 1-14 - Docker Kill](1-14-Docker-Kill.md)
+-
```

Comment:
Minor: would it be possible to shrink the heading size for this list? The title is fine, but each list item seems jarringly large if you're used to looking at markdown on github. :)